infrastructure/code/issue_enhancer.go

```diff
@@ -187,15 +189,18 @@ func (b *IssueEnhancer) autofixFunc(ctx context.Context, issue snyk.Issue,
 		defer pollingTicker.Stop()
 		timeoutTimer := time.NewTimer(2 * time.Minute)
 		defer timeoutTimer.Stop()
+		tries := 1.0
 		for {
 			select {
 			case <-timeoutTimer.C:
 				log.Error().Str("method", "GetAutofixSuggestions").Str("requestId", b.requestId).Msg("timeout requesting autofix")
 				b.notifier.SendShowMessage(sglsp.MTError, "Something went wrong. Please try again. Request ID: "+b.requestId)
 				return nil
 			case <-pollingTicker.C:
+				p.ReportWithMessage(int(math.Min(tries, 99)), "Polling for fix...")
 				fix, complete := pollFunc()
 				if !complete {
+					tries++
```

Comment:
I think this is going it be showing 99 until tries passes 99, right? We are not aiming to show a progress I understand

Reply:
Now, it's gonna show a progress up to the 99s limit - with Max it was only showing 99...